### PR TITLE
CopyContentToField support list value

### PR DESCRIPTION
**Pull request by erezh31**

### CODE REVIEW

File: Scripts/CopyContextToField/CopyContextToField.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+args:
+- default: false
+  description: The context key to get the value from.
+  isArray: false
+  name: sourceContextKey
```
Comment (anara123) [inline Scripts/CopyContextToField/CopyContextToField.yml]: our convention for arg names is snake case.
sourceContextKey => source_context_key

**Comment by Itay4** [comment]
@erezh31 whats our status here?

**Comment by erezh31** [comment]
I will fix it

File: Scripts/CopyContextToField/CopyContextToField.yml (hunk 1 above)
Comment (erezh31) [reply Scripts/CopyContextToField/CopyContextToField.yml]: Sure? I know for integrations, but most of the automation it's not, at least all the automations I wrote...

**Comment by erezh31** [comment]
@anara123 I commented \ fixed the review

**Comment by anara123** [comment]
@kirbles19 please review
https://github.com/demisto/content/blob/copy_context_fix/Scripts/CopyContextToField/CopyContextToField.yml
https://github.com/demisto/content/blob/copy_context_fix/Scripts/CopyContextToField/CHANGELOG.md

**Comment by kirbles19** [comment]
@erezh31 All done.

**Comment by Itay4** [comment]
@erezh31 whats our status here?

**Comment by Itay4** [comment]
@erezh31 ?

**Pull request merged**